osu.Game.Rulesets.Osu/Difficulty/Skills/Speed.cs

```diff
@@ -79,6 +81,9 @@ protected override double StrainValueOf(DifficultyHitObject current)
                 }
             }
 
+            if (Mods.Any(m => m is OsuModRelax))
+                speedBonus = 0.0;
```

Comment:
If you do this every object resolves to `0.2375 * angleBonus` , distance scaling is always zero. What is the intention here?

Returning zero for speed in PerformanceCalculator could be more straightforward, or you could return `angleBonus * Math.Pow(distance / single_spacing_threshold, 3.5)` here if you want the 'aiming' part of speed counted towards pp.